src/components/NcActionInput/NcActionInput.vue

```diff
@@ -85,22 +86,32 @@ For the multiselect component, all events will be passed through. Please see the
 
 				<template v-else>
 					<input :id="id" type="submit" class="action-input__submit">
-
-					<input :type="type"
-						:value="value"
-						:placeholder="text"
-						:disabled="disabled"
-						:aria-label="ariaLabel"
-						v-bind="$attrs"
-						:class="{ focusable: isFocusable }"
-						class="action-input__input"
-						@input="onInput"
-						@change="onChange">
-					<!-- allow the custom font to inject a ::before
-						not possible on input[type=submit] -->
-					<label v-show="!disabled" :for="id" class="action-input__label">
-						<ArrowRight :size="20" />
-					</label>
+					<div class="action-input__input-label-container">
```

Comment:
Should we probably create a more-understandable html structure, because now label ``` action-input__text-label ``` belongs to input ``` action-input__input-container ``` which is inside of ``` action-input__input-container ```. 

And input ``` action-input__submit``` belongs to label ``` action-input__label ``` which is inside of ```action-input__input-label-container``` and  ```action-input__input-label-container```. 

A bit confusing i think

Reply:
*if it works we can leave it like this for a day we have more time

Reply:
Good point @JuliaKirschenheuter! A refactor of the HTML structure here would be out-of-scope so we can do this in a separate PR